R/utils.R

```diff
@@ -836,8 +850,24 @@ getStackTrace <- function(expr, debug = FALSE, prune_errors = TRUE) {
     )
     } else {
       evalq(expr)
+  }
+}
+
+getLineWithError <- function(currentCall, formatted=TRUE) {
+  srcref <- attr(currentCall, "srcref", exact = TRUE)
+  if (!is.null(srcref) & !(getAppPath()==FALSE)) {
+    # filename
+    srcfile <- attr(srcref, "srcfile", exact = TRUE)
+    # line number
+    if (formatted) {
+      crayon::yellow$italic(sprintf('Line %s in %s', srcref[[1]], srcfile$filename))
+    } else {
+      sprintf('Line %s in %s', srcref[[1]], srcfile$filename)
```

Comment:
I'd reverse the order, and to make it easier to scan particularly in the unformatted case add a separator... something like `sprintf('-- %s, Line %s', srcfile$filename, srcref[[1]])`

Also 🌴 should only need `sprintf` once

Reply:
fixed in ff632ce